### ISSUE TITLE
AttributeError: 'Mesh' object has no attribute 'use_customdata_edge_bevel'

### ISSUE THREAD

**Issue by Celestial-Vixen** [TobyLobster/ImportLDraw#85]
I am getting this issue when trying to import using Blender 3.6.0 and a fresh install of ldraw parts.

```
Python: Traceback (most recent call last):
  File "C:\Users\Kristy Work\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\ImportLDraw-master\importldraw.py", line 391, in execute
    loadldraw.Options.overwriteExistingMeshes    = False
  File "C:\Users\Kristy Work\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\ImportLDraw-master\loadldraw\loadldraw.py", line 4300, in loadFromFile
    # In practice, this is not completely accurate, since the perspective projection changes the objects silhouette in 2d space
  File "C:\Users\Kristy Work\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\ImportLDraw-master\loadldraw\loadldraw.py", line 3683, in createBlenderObjectsFromNode
    bpy.ops.object.mode_set(mode='EDIT')
  File "C:\Users\Kristy Work\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\ImportLDraw-master\loadldraw\loadldraw.py", line 3579, in createBlenderObjectsFromNode
    if Options.addBevelModifier:
AttributeError: 'Mesh' object has no attribute 'use_customdata_edge_bevel'
```
I tried disabling the "Add Bevel" option, but the same issue is returned.

**Comment by darkantonsg** [comment]
Same error and behaviour for me. Any idea how to go around it?

**Comment by Celestial-Vixen** [comment]
> Same error and behavior for me. I imported the LDR file in Blender 3.3, saved as Blend file and then re-opened in Blender 3.6

Hi! I think I eventually found a fix for this but my memory is poor, it may have been a different plugin around the same time. The newest plugin is working on blender 3.6.5 for me, so I can't trouble shoot it again. 

If this WAS the one of two plugins I was having an issue with that I fixed, I had to edit the plugin to do so. Will update if I  can find the fix.
